install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3388,7 +3388,12 @@ clustermesh:
 
   # -- Annotations to be added to all top-level clustermesh objects (resources under templates/clustermesh-apiserver and templates/clustermesh-config)
   annotations: {}
-  # -- Clustermesh explicit configuration.
+  # -- Clustermesh explicit configuration. This can be disabled to be able to
+  # generate the following secrets yourself:
+  # - cilium-clustermesh (used by cilium-agent/cilium-operator to connect to
+  #   the local etcd if KVStoreMesh is enabled or remote clusters etcd if KVStoreMesh
+  #   is disabled)
```

Comment:
s/the local etcd/the local etcd instance/
s/remote clusters etcd/the remote (etcd) clusters/